Integrate price feeder

zarazan opened the pull request:
- Added functionality to the start-chain bash script that creates a new price-feeder service alongside any umeed service that is created.
- Each new price-feeder service has a number associated with it just like the umeed services that are created where umeed-1 is the node that umeed-1-pf sends its price transactions to.

zarazan committed:
WIP Add price feeder services

zarazan committed:
WIP Add price-feeder start chain updates

zarazan committed:
Move price-feeder config to template

- Add some sed commands for Keith

adamewozniak commented on scripts/chain/start_chain.sh:
```diff
@@ -235,3 +235,53 @@ do
+    PF_CONFIG="${CONFIG_DIR}/price-feeder.toml"
+    cp ../configs/price-feeder.toml $CONFIG_DIR
+
+    UMEE_E2E_PRICE_FEEDER_VALIDATOR=$(eval "umeed keys show validator${a} --home ${DAEMON_HOME}-${a} --bech val --keyring-backend test --output json | jq .address")
```
Let's remove the UMEE_E2E parts of these environment variables:
```suggestion
    PRICE_FEEDER_VALIDATOR=$(eval "umeed keys show validator${a} --home ${DAEMON_HOME}-${a} --bech val --keyring-backend test --output json | jq .address")
```

zarazan replied:
Sounds good, they are a little long. I'll update them in the price-feeder config as well.

adamewozniak commented on scripts/chain/start_chain.sh:
```diff
@@ -235,3 +235,53 @@ do
+    cp ../configs/price-feeder.toml $CONFIG_DIR
+
+    UMEE_E2E_PRICE_FEEDER_VALIDATOR=$(eval "umeed keys show validator${a} --home ${DAEMON_HOME}-${a} --bech val --keyring-backend test --output json | jq .address")
+    UMEE_E2E_PRICE_FEEDER_ADDRESS=$(eval "umeed keys show validator${a} --home ${DAEMON_HOME}-${a} --bech acc --keyring-backend test --output json | jq .address")
```
```suggestion
    PRICE_FEEDER_ADDRESS=$(eval "umeed keys show validator${a} --home ${DAEMON_HOME}-${a} --bech acc --keyring-backend test --output json | jq .address")
```

zarazan replied:
https://github.com/umee-network/qa-integration/pull/9/commits/556403f5d3f9f8eb7b6e5a38d88a8d720473f01b

adamewozniak reviewed (approved):
just one small nit but lgtm! 🥳

adamewozniak committed:
Merge branch 'master' into zarazan/integrate-price-feeder

zarazan committed:
Make price feeder variables shorter

zarazan committed:
Re-remove go check

I'll fix the go install in the env-check file in another PR - keep it on the version that work on github for now.

zarazan committed:
Merge branch 'master' into zarazan/integrate-price-feeder
